imports/api/logic/grad-checker/faculty/computing/computer-science/AY-1617/breadth-depth/team-project/teamProject.test.js

```diff
@@ -0,0 +1,57 @@
+import { assert, expect } from 'meteor/practicalmeteor:chai';
```

Comment:
Just wanna say here that the detailed variable names are helpful in understanding the code as you glance thru!